compiler/rustc_target/src/spec/aarch64_apple_ios_sim.rs

```diff
@@ -21,7 +21,7 @@ pub fn target() -> Target {
             // These arguments are not actually invoked - they just have
```

Comment:
This comment is no longer accurate. It's also not clear to me why need to change it for the `-simulator` target.

Reply:
My apologies! It should not be in this PR coz it is for simulators.